op-tests/test-blspy-g2.txt

```diff
@@ -0,0 +1,120 @@
+; This file was generated by tools/generate-bls-tests.py
+
+bls_g2_add 0x96bbcfbca241004ba03968b7433664c1b944245828af7a1ebda771495c90d4816508d65db40b601bce51a3689c251f731699411b56e2172dd82eb96f571a691ba2984f1e4e39a05646ceb9c82d1108e78fe3fdfd94cff4fb447efe498dcecc25 0xa35068bd876cd804a57e9695ad9585271b20f2a67c0f70a05ec47c597a6c3754083d1a974b6bf9a90c1e4e5208b73d7a120ce4190f328a3b90a15c2c7a7973a5894a8e16a63c36adb4e4995d165192d9b8c26fc15a84655c90a92b846f7952fa => 0x9666a3ff8474a9c084fe71d8ad1107f6e89e1bf9fbff0d6e0f20507e79ef4e35783fca3fedbb8f0d0e7f4434b5df37631170fe5be0096fdc72183b2de89aed593060bbc8700df230329afdb9020333007955c4e405a5776f6df81cb39ca55af4 | 11135562
```

Comment:
Test with 0 and 1 parameter.

Reply:
the case of no arguments is covered here: https://github.com/Chia-Network/clvm_rs/pull/274/commits/83ff075a5957ebb89fa5f64ff9fe9aaf208b3aa3#diff-8e926fa1adc34a468edbbfcb76a3996682c288dc76919bad5a6876cbe68fde84R66

I'll add the single argument case next to it